Client/Frontend/Browser/UserScriptManager.swift

```diff
@@ -252,7 +252,7 @@ class UserScriptManager {
         return WKUserScript(source: source,
                             injectionTime: .atDocumentStart,
                             forMainFrameOnly: false,
-                            in: .defaultClient)
+                            in: .page)
```

Comment:
Just double checking is there a handler we need to also set as `page` or just the `WKUserScript`?

Reply:
We have no handler for this script. Usually it is true that `both` need it, but we're good here :)

It doesn't call any handlers :D
```
if (window.MediaSource || window.WebKitMediaSource || window.HTMLMediaElement && HTMLMediaElement.prototype.webkitSourceAddId) {
    
    window.MediaSource = null;
    window.WebKitMediaSource = null;
    //HTMLMediaElement.prototype.webkitSourceAddId = null;
    //window.SourceBuffer = null;
    
    delete window.MediaSource;
    delete window.WebKitMediaSource;
}
```